packages/mediapipe-core/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: mediapipe_core
+description: A starting point for Dart libraries or applications.
+version: 1.0.0
```

Comment:
The native assets feature is in experimental (and has not landed yet in Flutter for Android), anyone trying to get this package with the feature not yet enabled cannot use it. So please be careful with publishing a package.

Consider adding `publish_to: none` for now to prevent accidentally publishing something.

Also, should probably be 0.something and/or `-wip`.

Reply:
Done!